apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx

```diff
@@ -138,6 +139,13 @@ const WorkflowEditorPage = ({
                     })}
                   </NavSection>
                 </When>
+                <When truthy={selectedChannel === ChannelTypeEnum.DIGEST}>
```

Comment:
I just love this When component 😃

Reply:
I have a more advanced in store for `When` we need it 😉